zerver/lib/test_runner.py

```diff
@@ -48,20 +47,36 @@ def get_test_method(test):
     # type: (TestCase) -> Callable[[], None]
     return getattr(test, test._testMethodName)
 
+# Each tuple is delay, test_name, slowness_reason
+TEST_TIMINGS = [] # type: List[Tuple[float, str, str]]
+
+
+def report_slow_tests():
```

Comment:
Thx.  You can see I was thinking of it in one place, but I need to get to get in the habit of doing it everywhere.